docarray/array/storage/elastic/getsetdel.py

```diff
@@ -50,6 +54,41 @@ def _get_doc_by_id(self, _id: str) -> 'Document':
         """
         return self._getitem(_id)
 
+    def _get_docs_by_ids(self, ids: Sequence[str]) -> Iterable['Document']:
+        """Concrete implementation of base class' ``_get_docs_by_ids``
+
+        :param ids:  ids of the document
+        :return: Iterable[Document]
+        """
+        accumulated_docs = []
+        accumulated_docs_not_found = []
+
+        if len(ids) < 1:
+            return accumulated_docs
+
+        # Handle if doc len is more than MAX_ES_RETURNED_DOCS
+        nrof_arr_split = math.ceil(len(ids) / MAX_ES_RETURNED_DOCS)
+        split_ids = np.array_split(ids, nrof_arr_split)
+        for split in split_ids:
+            es_docs = self._client.mget(index=self._config.index_name, ids=split)[
+                'docs'
+            ]
+
+            docs = [
+                Document.from_base64(doc['_source']['blob'])
+                for doc in es_docs
+                if doc['found'] is True
+            ]
+            docs_not_found = [doc['_id'] for doc in es_docs if doc['found'] is False]
```

Comment:
```suggestion
            docs_id_not_found = [doc['_id'] for doc in es_docs if doc['found'] is False]
```
lets rename it to show that it is the ids and not the docs directly